core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -69,19 +69,6 @@ public static function substr($strString, $intNumberOfChars, $strEllipsis=' …'
 			break;
 		}
 
-		if ($strEllipsis === false)
-		{
-			trigger_deprecation('contao/core-bundle', '4.0', 'Passing "false" as third argument to "Contao\StringUtil::substr()" has been deprecated and will no longer work in Contao 5.0. Pass an empty string instead.');
-			$strEllipsis = '';
-		}
-
-		// Deprecated since Contao 4.0, to be removed in Contao 5.0
-		if ($strEllipsis === true)
-		{
-			trigger_deprecation('contao/core-bundle', '4.0', 'Passing "true" as third argument to "Contao\StringUtil::substr()" has been deprecated and will no longer work in Contao 5.0. Pass the ellipsis string instead.');
-			$strEllipsis = ' …';
-		}
-
```

Comment:
Simply removing this without adding a type hint that enforces the type will not work IMHO. Maybe we should throw an `InvalidArgumentException` if the argument `is_bool()`?